js/coinsbit.js

```diff
@@ -0,0 +1,742 @@
+                'setLeverage': undefined,
+                'withdraw': undefined,
+            },
+            'timeframes': {
+                '1m': 60, // works for max 12 hours diapason (between from-to)
+                '30m': 1800, // for max 24 hours diapason
+                '1h': 3600, // for max 7 days diapason
+                '1d': 86400, // for any diapason
+            },
+            'urls': {
+                'logo': '  <<< TODO >>>   https://upload.wikimedia.org/wikipedia/commons/8/8c/Coinsbit.png  ',
+                'api': {
+                    'spot': {
+                        'public': 'https://coinsbit.io/api/',
+                        'private': 'https://coinsbit.io/api/',
+                    },
+                },
+                'www': 'https://coinsbit.io/',
+                'doc': [
+                    'https://www.notion.so/coinsbitwsapi/API-COINSBIT-WS-API-COINSBIT-cf1044cff30646d49a0bab0e28f27a87',
+                    'https://github.com/Coinsbit-connect/api/wiki/API',
+                ],
+                'fees': [
+                    'https://coinsbit.io/fee-schedule',
+                ],
+                'referral': '  <<< TODO >>>   ',
+            },
+            'api': {
+                'spot': {
+                    'public': {
+                        'get': {
+                            'markets': 1,
+                            'tickers': 1,
+                            'ticker': 1,
+                            'book': 1,
+                            'history': 1,
+                            'history/result': 1,
+                            'products': 1, // Of no use, everything already available in 'markets'
+                            'symbols': 1, // Of no use, everything already available in 'markets'
+                            'depth/result': 1,
+                            'kline': 1,
+                        },
+                    },
+                    'private': {
+                        'get': {
+                        },
+                        'post': {
+                            'order/new': 1,
+                            'order/cancel': 1,
+                            'orders': 1,
+                            'account/balances': 1,
+                            'account/balance': 1,
+                            'account/order': 1,
+                            'account/trades': 1,
+                            'account/order_history': 1,
+                            'account/order_history_list': 1,
+                        },
+                    },
+                },
+            },
+            'precisionMode': TICK_SIZE,
+            'fees': {
+                'trading': {
+                    'tierBased': false,
+                    'percentage': true,
+                    'maker': 0.2 / 100,
+                    'taker': 0.2 / 100,
+                },
+            },
+            'options': {
+                'defaultType': 'spot', // spot, ..
```

Comment:
Do they actually have swaps, futures, contracts, options or something like that? Or do they only have spot markets?

Reply:
in API they don't have other types, but on site/UI, there are otc, dex ... So, I thought as a placeholder to set that there too.
Should it be removed?

Reply:
If we're not adding contracts to it right now in this specific PR – let's remove it.

Reply:
Let's first check if we should add contracts though.

Reply:
Looks like they don't have non-spot markets at this time.

Reply:
done